lldb/source/Core/ModuleList.cpp

```diff
@@ -714,6 +714,8 @@ uint32_t ModuleList::ResolveSymbolContextsForFileSpec(
     const FileSpec &file_spec, uint32_t line, bool check_inlines,
     SymbolContextItem resolve_scope, SymbolContextList &sc_list) const {
   std::lock_guard<std::recursive_mutex> guard(m_modules_mutex);
+  // If we're looking for a header (not source), then need to check inline.
+  check_inlines = check_inlines || !file_spec.IsSourceImplementationFile();
```

Comment:
I don't think this should be done here. The breakpoint setting code also goes through this code path and it works fine without it. I suggest looking at how that works and emulating it. I suspect it involves looking at the `target.inline-breakpoint-strategy` setting. I think it'd make sense to do that here as well (if anyone disagrees with that, do chime in), even though the setting is called inline-*breakpoint*-strategy, as the setting is really about where we go to search when looking up a file.

Reply:
Done! Thanks for the pointer! :)